cmd/wait-for-files-to-exist/main.go

```diff
@@ -35,85 +36,120 @@ func main() {
 	}
 	flag.Parse()
 
-	if flag.NArg() < 1 {
-		exit("must specify a file to watch")
-	}
-	if flag.NArg() > 1 {
-		exit("only one file can be specified")
-	}
-	fileToWaitFor := flag.Arg(0)
-
+	// Set up a timeout context if requested.
 	if *timeout > 0 {
 		timeoutCtx, cancel := context.WithTimeout(ctx, *timeout)
 		defer cancel()
 
 		ctx = timeoutCtx
 	}
 
-	w, err := fsnotify.NewWatcher()
-	if err != nil {
-		exit("creating a new watcher: %s", err)
+	if flag.NArg() < 1 {
+		exit("must specify at-least one file to watch")
 	}
-	defer w.Close()
+	fileToWaitFor := flag.Args()
 
-	// Start listening for events.
-	errChan := make(chan error)
-	go fileLoop(ctx, w, fileToWaitFor, errChan)
+	// Validate and exit early if any file is invalid.
+	for _, f := range fileToWaitFor {
+		if err := validateFileToWaitFor(f); err != nil {
+			exit("%w", err)
```

Comment:
`%w` only makes sense when using `fmt.Errorf` iirc.

```suggestion
			exit("%v", err)
```

Reply:
Oops, I saw the the failure in the linter in one of the `fmt.Errorf` and incorrectly changed all of them